terraform/autospotting/variables.tf

```diff
@@ -1,4 +1,14 @@
 # Autospotting configuration
+variable "autospotting_disallowed_instance_types" {
+  description = <<EOF
+Comma separated list of disallowed instance types for spot requests,
+in case you want to exclude specific types (also support globs).
+
+Example: 't2.*,m4.large'
```

Comment:
Are these supposed to be supported?

Please create some test cases for the globs as well.

Reply:
No, I meant to ask if we do support glob matching for the black/white-listing logic. If so, we should have a few test cases for this.

Reply:
Never mind, I saw it now.